### PR TITLE
Codeql of portus

### PR DESCRIPTION
**Pull request by swetatech24**
https://github.com/Be-Secure/BeSLighthouse/issues/146 portus codeql

### CODE REVIEW

**Review by harimohanr — CHANGES_REQUESTED** [review]
@swetatech24 There seems to be some text present in the beginning of the portus codeql report. please remove those and make sure the codeql report is in correct format.

**Comment by swetatech24** [comment]
I guess now the issue is resolved.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Review by asa1997 — CHANGES_REQUESTED** [review]
@swetatech24 Wrong file for grav codeql report

File: grav/2.4.3/sast/Portus-2.4.3-codeql-report.json [hunk 1]
```diff
@@ -0,0 +1,740 @@
+ % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
```
Comment (asa1997) [inline grav/2.4.3/sast/Portus-2.4.3-codeql-report.json:1]: @swetatech24 Please take these out.

**Review by harimohanr — CHANGES_REQUESTED** [review]
@swetatech24 You have uploaded a wrong codeql file for grav. The file you have uploaded is for Portus.

**Pull request closed without merging**